modules/student/examples/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, we already have a pom file for all the examples, is there a reason we need another one?

Reply:
Yes, the tutorial uses a pom file to get the dependencies so folk can get up and running quickly. I put it here just to make sure I have a copy of it outside of the docs.

Reply:
Ok, just thinking whenever we have a new SDK release this pom will need to be updated as well to make sure we always keep the Couchbase SDK version up to date.

For example, the latest SDK version is now 3.2.2...so that will need to be updated in this pom also.
https://github.com/couchbase/docs-sdk-java/pull/290/files/b7424a79e0b06d1932c6344474e9035195701ed3#diff-61cf07667ead974cd5068f21e38dd826a42d73f5f60301719dd237ec0a647914L11-R13

Reply:
Done!

Yup, not ideal, but I'm keen that they just have the smallest number of files to deal with to get up and running.

Reply:
Can we generate the pom files in antora... 🤔